inc/shortcodes.php

```diff
@@ -18,17 +18,20 @@ function uds_wordpress_shortcode_sidebar_menu( $atts ) {
 		array(
 			'menu'            => '',
 			'title'           => '',
+			'id'              => 'sidebar-menu',
 		),
```

Comment:
The id cannot be static, just in case if we have two sidebars in one page.

Reply:
Updated this with the following:

- Removed my custom filter for the 'is-active' class, and modified Zainab's existing code
- Removed the 'id' argument from the short code and used the menu's name as the ID, since the name is already available and is a perfect place to craft a custom name for a menu!